UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -340,20 +344,24 @@ public void BodyPartRemoveHealthMaster()
 		/// <summary>
 		/// Server only - Tries to remove a body part
 		/// </summary>
-		public void TryRemoveFromBody(bool beingGibbed = false)
+		public void TryRemoveFromBody(bool beingGibbed = false, bool CausesBleed = true, bool Destroy = false, bool PreventGibb_Death = false)
```

Comment:
This entire change seems like it needs to be logged or explained.

Reply:
What would you add?

Reply:
This change can disable bleeding on some parts, destory them and can prevent death on gib; correct?

Reply:
yeah?